copyshop.py

```diff
@@ -4,17 +4,34 @@
 
 import argparse
 import os
+import re
+import json
 import requests
 
 USERNAME = os.environ["SAUCE_USERNAME"]
 ACCESS_KEY = os.environ["SAUCE_ACCESS_KEY"]
+MASTER_PASSWORD = os.environ["MASTER_PASSWORD"]
```

Comment:
We should probably use os.environ.get() instead.
If you don't have any of these env variables in your machine, the file throws an exception:

```bash
(tmp-c808c3c68e74c4cd) ➜  copy-shop git:(RDC) ✗ copyshop
Traceback (most recent call last):
  File "/Users/enriquegonzalez/.virtualenvs/tmp-c808c3c68e74c4cd/bin/copyshop", line 11, in <module>
    load_entry_point('copyshop', 'console_scripts', 'copyshop')()
  File "/Users/enriquegonzalez/.virtualenvs/tmp-c808c3c68e74c4cd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Users/enriquegonzalez/.virtualenvs/tmp-c808c3c68e74c4cd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/Users/enriquegonzalez/.virtualenvs/tmp-c808c3c68e74c4cd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/Users/enriquegonzalez/.virtualenvs/tmp-c808c3c68e74c4cd/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/enriquegonzalez/git/copy-shop/copyshop.py", line 13, in <module>
    MASTER_PASSWORD = os.environ["MASTER_PASSWORD"]
  File "/Users/enriquegonzalez/.virtualenvs/tmp-c808c3c68e74c4cd/bin/../lib/python3.7/os.py", line 678, in __getitem__
    raise KeyError(key) from None
KeyError: 'MASTER_PASSWORD'
```

Reply:
Depending on the type of report you're cloning, these variables are necessary, so I think we should have a slightly smarter validation.
But yes, that's a very good point, I will fix that.

Reply:
Added a quickfix as suggested.